### DISCUSSION TITLE
Skybox 변경

### DISCUSSION THREAD

**Discussion by peterkangdev** [naverz/zepeto-studio-kor#487] [5. Q&A]
다음과 같이 대입하면 에러가 나던데, 

RenderSettings.skybox = this.skybox;

<img width="643" alt="스크린샷 2022-03-13 오후 6 05 08" src="https://user-images.githubusercontent.com/30133892/158052639-786c5a2b-9b5f-44f5-80a0-7dee52293b72.png">

skybox 를 변경할 수 없나요?
제가 모르는 부분이 또 있을까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. 사용하려는 스카이박스의 구조(아래 스크린샷 참고)와 전체 코드를 보면 더 자세히 알 수 있을 것 같습니다.
코드 말고 인스펙터 상에서 적용하는 방식은 잘 되는지도 확인해 보세요.
![Inspector-Skybox](https://user-images.githubusercontent.com/100118650/158090812-bffe475a-b94b-4753-bb86-1f75dbcfa0bf.jpeg)
